### DISCUSSION TITLE
On Parallels (Windows 11 emulator) running quasar dev -m electron fails as it tries to launch the Mac version

### DISCUSSION THREAD

**Discussion by dannyconnell** [quasarframework/quasar#15551] [CLI - Electron mode] [answered]
I have a Mac and I'm using Parallels to test the Windows version of my app.

I can build the app for Windows (by running `quasar build -m electron` on the Mac) and test it (on Parallels) with no problems.

But I need to launch the app in dev mode (on Parallels) so that I can see main process errors.

But (in Parallels > Windows 11 > Powershell) when I run: 

`quasar dev -m electron`

It fails with this output:

```
 App • Running "Z:\Projects\fudget2\node_modules\electron\dist\Electron.app\Contents\MacOS\Electron --inspect=5858 Z:\Projects\fudget2\.quasar\electron\electron-main.js" in Z:\Projects\fudget2

'Z:\Projects\fudget2\node_modules\electron\dist\Electron.app\Contents\MacOS\Electron' is not recognized as an internal or external command,
operable program or batch file.

 App • Command "Z:\Projects\fudget2\node_modules\electron\dist\Electron.app\Contents\MacOS\Electron" failed with exit code: 1

 App • ⚠️  Electron process ended with error code: 1
```

You can see that it is trying to launch the Mac version of the app (and not the Windows version).

It seems like I need to, either, trick Parallels into believing it is actually real-life Windows, or perhaps there is a way in Quasar of specifying / forcing the platform that will be launched with the `quasar dev -m electron` command?

Thanks in advance for any help or ideas! 👍

**Comment by pdanpdan** [comment, marked as answer]
While in emulator you need to npm install/yarn/whatever in order to get the modules for that platform

**Reply by dannyconnell** [reply]
So basic 🤦‍♂️. Thanks, @pdanpdan ! 👍
